fastplotlib/graphics/selectors/_linear.py

```diff
@@ -227,11 +229,19 @@ def __init__(
             axis=axis,
         )
 
+    @property
+    def selection(self) -> LinearSelectionFeature:
+        """
+        The current selection in world space. These bounds will NOT necessarily correspond to the
+        indices of the data that are under the selection.
```

Comment:
this is not bounds, reword to be relevant to linear selectors

Reply:
They should use `get_selected_index()` to get data index that corresponds to the selection. It maps from graphic world space to data space.